guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc

```diff
@@ -0,0 +1,62 @@
+[[migrating_pulp_content]]
+
+= Migrating Content to Pulp 3
+
+The time to prepare content for Pulp 3 depends on the amount of content.
+For large systems, this can mean several days of downtime.
+
+Use this procedure to begin migrating content from Pulp 2 to Pulp 3.
```

Comment:
It might be good to mention that:
* the content migration happens on 6.9, prior to upgrading to 6.10
* The content migration can be run online, but cpu, disk, and memory resources will be used during that time, and operations around repository syncing and content view publishing may take longer
* As part of the upgrade to 6.10 the satellite will do a final migration and then switchover to pulp 3

Reply:
Added.

Reply:
Included